lib/configValidator.js

```diff
@@ -0,0 +1,60 @@
+var Ajv = require('ajv');
+
+const SUPPORTED_METHODS = ['get', 'post', 'put', 'delete', 'patch'];
+
+const responseSchema = {
+  '$id': 'Response.json',
+  'resposeDef': {
+    'type': 'object',
+    'properties': {
+      'status': {
+        'type': 'integer'
+      },
+      'headers': {
+        'type': 'object',
+        'propertyNames': {
+          'pattern': '^[\\w-]+$'
+        },
+        'patternProperties': {
+          '[\\w-]+': { 'type': 'string' }
+        }
+      },
+      'data': {
+        'type': 'object'
+      }
+    },
+    'additionalProperties': false
+  }
+};
+
+const schema = {
+  '$id': 'ResponseDefs.json',
+  'oneOf': [
+    {
+      '$ref': 'Response.json#/resposeDef'
+    },
+    {
+      'type': 'array',
+      'items': {
+        '$ref': 'Response.json#/resposeDef'
+      }
+    }
+  ]
+};
+
+module.exports = (responseConfig, resourcePath, method) => {
+  if (!SUPPORTED_METHODS.includes(method)) {
+    throw new Error(`Method ${method} is not supported.`);
+  }
+  var ajv = new Ajv({ schemas: [schema, responseSchema] });
```

Comment:
var?

Reply:
Whoops, yes, left-over from the example snippet. I'll get that in a future change.